### ISSUE TITLE
Reviewing stale branches

### ISSUE THREAD

**Issue by MichaIng** [HTPC-Manager/HTPC-Manager#36]
Little review cleanup makes sense:
- https://github.com/HTPC-Manager/HTPC-Manager/tree/cardigan: The project is [basically dead](https://github.com/cardigann/cardigann/graphs/code-frequency), so I suggest to remove the branch. When it is resurrected, the branch can be restored from base branches.
- Kodi/VLC web player: https://github.com/HTPC-Manager/HTPC-Manager/compare/master2...HTPC-Manager:video_stream_temp
    Much effort has gone into this one. As it's more than 6 years old, I just fear that used technologies are outdated, APIs changed etc. Would be interesting to test, but I think, if wanted, it is less work to start from scratch, using modern Kodi and/or VLC API, HTML5 etc, ignoring old browser support etc.
- Fixes from the old [master](https://github.com/HTPC-Manager/HTPC-Manager/compare/master2...HTPC-Manager:master) branch have been applied to master2 already, so that one can be removed.

So in short, I suggest to remove those three branches so that we have master2 only. An idea would be to create some sorts of dev branch, where we collect smaller commits for a while and release them after another review/testing on a regular basis, so that users don't see an update on every little change and e.g. the Docker container build server is not overloaded with rebuilds for a single little commit only 😄: https://github.com/linuxserver/docker-htpcmanager
Currently I aim to merge multiple PRs at once, when some more accumulated and having them open for review and testing for a while, so a salve of multiple commits work a little like a release 🙂.

**Comment by gmiranda** [comment]
Nice idea.  I agree with removing the cardigan and old master branches (the latter is confusing and we basically inherited it).

As for video_stream_temp, it sounds incredible, but on one hand it's too dated (our current code might not be compatible) and on the other hand it seems to be really big (code lines) as well as including code from other projects. I'm confident that the Kodi API is somewhat stable, but I don't think it's worth the effort unless you have the spare time. Certainly I don't :disappointed:

**Comment by MichaIng** [comment]
Okay, let's leave the video branch in place for now. Probably someone find the time to pick it up, even when just using it for spare parts to start this feature from scratch 🙂.

**Comment by MichaIng** [comment]
Old branches removed, and opened a PR + rebased the video player branch: #45
Was not too hard to resolve conflicts, pretty clean, of course no guarantee that the additional modules, the included library and the imports are Python 3 compatible, but it serves as basis to test and in case continue the effort that has been put into it.

I'll mark this issue hence as closed.
